tests/configuration.py

```diff
@@ -32,681 +32,35 @@
-                "5G-Station-Name": "wlan0",
-                "AX-Station-Name": "ax"
-            }
-        }
-    },  # checked   deployed    1.x
-    "basic-03": {
-        "controller": {
-            'url': 'https://sec-qa01.cicd.lab.wlan.tip.build:16001',  # API base url for the controller
-            'username': "tip@ucentral.com",
-            'password': 'OpenWifi%123',
-        },
-        'access_point': [
-            {
-                'model': 'tp-link_ec420-g1',
-                'mode': 'wifi5',
-                'serial': '001122090801',
-                'jumphost': True,
-                'ip': "localhost",
-                'username': "lanforge",
-                'password': "pumpkin77",
-                'port': 8723,
-                'jumphost_tty': '/dev/ttyAP3',
-                'version': "next-latest"
-            }
-        ],
-        "traffic_generator": {
-            "name": "lanforge",
-            "details": {
-                "ip": "localhost",
-                "port": 8722,
-                "ssh_port": 8724,
-                "2.4G-Radio": ["1.1.wiphy4"],
-                "5G-Radio": ["1.1.wiphy5"],
-                "AX-Radio": ["1.1.wiphy0", "1.1.wiphy1", "1.1.wiphy2", "1.1.wiphy3"],
-                "upstream": "1.1.eth2",
-                "upstream_subnet": "10.28.2.1/24",
-                "uplink": "1.1.eth3",
-                "2.4G-Station-Name": "sta00",
-                "5G-Station-Name": "sta10",
-                "AX-Station-Name": "ax"
-            }
-        }
-    },  # checked   2.x
-    "basic-03a": {
-        "controller": {
-            'url': 'https://sec-qa01.cicd.lab.wlan.tip.build:16001',  # API base url for the controller
-            'username': "tip@ucentral.com",
-            'password': 'OpenWifi%123',
-        },
-        'access_point': [
-            {
-                'model': 'indio_um-305ac',
-                'mode': 'wifi5',
-                'serial': '706dec0a8a79',
-                'jumphost': True,
-                'ip': "localhost",
-                'username': "lanforge",
-                'password': "pumpkin77",
-                'port': 8723,
-                'jumphost_tty': '/dev/ttyAP6',
-                'version': "release-latest"
-            }
-        ],
-        "traffic_generator": {
-            "name": "lanforge",
-            "details": {
-                "ip": "localhost",
-                "port": 8722,
-                "ssh_port": 8724,
-                "2.4G-Radio": ["1.1.wiphy4"],
-                "5G-Radio": ["1.1.wiphy5"],
-                "AX-Radio": ["1.1.wiphy0", "1.1.wiphy1", "1.1.wiphy2", "1.1.wiphy3"],
-                "upstream": "1.1.eth2",
-                "upstream_subnet": "10.28.2.1/24",
-                "uplink": "1.1.eth3",
-                "2.4G-Station-Name": "sta00",
-                "5G-Station-Name": "sta10",
-                "AX-Station-Name": "ax"
-            }
-        }
-      },  # checked   2.x
+import json
+file = open('lab_info.json')
```

Comment:
Please use a context manager to read the file:
```python
with open('lab_info.json', 'r') as f:
  data = json.load(f)
```

Reply:
Also, let's introduce an environment variable that allows to override lab info file path.